Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```
@@ -1,594 +1,354 @@
+  margin-top: 0.25em;
+  padding: calc(0.5 * var(--remotion-themed-padding));
+  display: flex;
+  align-items: center;
+  grid-row: 2;
 }
 
-tr.bocListDataRow td.bocListDataCell a:active
+div.bocList div.bocListTableBlock div.bocListNavigator > span
 {
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
+  margin-right: var(--remotion-themed-padding);
 }
 
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie tr.bocListDataRow td.bocListDataCell > a
+div.bocList div.bocListTableBlock div.bocListNavigator > span > input
 {
-  position: relative;
+  min-height: unset;
+  height: 16px;
+  padding: calc(0.5 * var(--remotion-themed-padding));
+  margin: calc(0.5 * var(--remotion-themed-padding)) var(--remotion-themed-padding);
+  text-align: right;
 }
 
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie tr.bocListDataRow td.bocListDataCell > a::before
+div.bocList div.bocListTableBlock div.bocListNavigator img
 {
-  content: "";
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
+  height: 16px;
+  width: 16px;
+  border: var(--remotion-themed-border-width) solid transparent;
+  border-radius: 2px;
+  padding: calc(0.5 * var(--remotion-themed-padding));
+  margin: calc(0.5 * var(--remotion-themed-padding));
+  background-color: white;
 }
 
-/*  Style applied to the disabled anchor tags during hovering in BocList's data rows.  */
-tr.bocListDataRow td.bocListDataCell a.disabled:hover,
-tr.bocListDataRow td.bocListDataCell a.disabled:active
+div.bocList div.bocListTableBlock div.bocListNavigator a[href] img
 {
-  color: Gray;
-  border: 1px solid transparent;
+  border-color: var(--color-standard-border);
 }
 
-/* Style applied to SPAN elements in command-enabled column cells */
-tr.bocListDataRow td.bocListDataCell span.CommandText
+div.bocList div.bocListTableBlock div.bocListNavigator a[href]:hover img
 {
-  vertical-align: middle;
+  border-color: var(--color-primary);
+  background-color: var(--color-highlight);
 }
 
-
-/* Style to override the BocList's anchor style for drop down menus rendered inside a cell. */
-td.bocListDataCell div ul.DropDownMenuOptions li a
+div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 {
-  display: block;
+  filter: opacity(0.5);
 }
+/*endregion*/
 
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a,
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:active,
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:focus,
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:hover
+/*region bocListMenuBlock*/
+div.bocList div.bocListMenuBlock
 {
-  background-color: #ECEEEC;
-  border-style: none;
-  padding: 2px;
-  color: gray;
+  grid-column: 2;
+  grid-row: 1;
+  background-color: var(--boclist-listmenu-background-color);
```

I think we can simplify this to 
```suggestion
  background-color: var(--color-contrast);
```
since it's only used once and we didn't declare variables for all concepts inside the controls.

---

`--boclist-tablehead-border-color` is also only used once, as well as `--boclist-horizontal-layout` and newly introduced `--boclist-menublock-minimum-width` and `--boclist-menublock-maximum-width` - should I inline all of them?

---

yes to the color.

no to the widths because of the idea to be able to override them via rendering a new value for the property or by overriding it in an application.

As for the horizontal layout. I'm not sure if that's a good extension point. probably inline?